README.rst

```diff
@@ -255,6 +292,39 @@ Example usage:
         }
     ]
 
+``OPENWISP_ADMIN_THEME_CSS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**default**: ``[]``
+
+Allows passing a list containing strings of URLs of custom CSS files.
+ Adding this setting does not override the default theme,
+ rather you will have to override, CSS selectors in your file.
+
+Example usage:
+
+.. code-block:: python
+
+    OPENWISP_ADMIN_THEME_CSS = [
+    	"http://127.0.0.1:8000/static/custom_admin_theme_css.css"
+	]
+	
+``OPENWISP_ADMIN_JS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**default**: ``[]``
+
+Allows to pass a list contaning strings of URLs of custom JS files.
+
+Example usage:
+
+.. code-block:: python
+
+    OPENWISP_ADMIN_THEME_CSS = [
+    	"../static/custom-admin-theme/js/you-custom-js1.css",
+    	"../static/custom-admin-theme/js/you-custom-js2.css",
+	]
```

Comment:
same here regarding indentation.

Also, use an absolute URL as in the CSS example (be consistent).

Reply:
I'd keep just one style for simplicity, you can add a comment to mention that both relative and absolute paths can be used